src/Transition/SupportFiles/Report Variables 9-2-0 to 9-3-0.csv

```diff
@@ -1,15 +1,17 @@
-9.2.0,9.3.0,Transition notes
-12,12,These numbers should be the number of report variables in the following list (including deletes). Two columns/numbers.
-old variable name,new variable name,-- add variable names (before this line) and leave off units -- <DELETE> to delete
-Water to Water Heat Pump Load Side Heat Transfer Rate,Heat Pump Load Side Heat Transfer Rate,
-Water to Water Heat Pump Load Side Heat Transfer Energy,EIR Heat Pump Load Side Heat Transfer Energy,

```

Comment:
Note these were in the wrong place and this line had a stray "EIR". That's been fixed here in the PR. @mitchute